_config.yml

```diff
@@ -22,18 +22,22 @@ title: Cryostat
 author: The Cryostat Authors
 email: cryostat-development@googlegroups.com
 description: >- # this means to ignore newlines until "baseurl:"
-  JDK Flight Recorder for Kubernetes
+  JDK Flight Recorder Management for Kubernetes and OpenShift
 baseurl: "" # the subpath of your site, e.g. /blog
 url: "https://cryostat.io" # the base hostname & protocol for your site, e.g. http://example.com
 #twitter_username: 
 github_username: cryostatio
+github_fork_url: "https://github.com/cryostatio/cryostat"
```

Comment:
Where is this used?

Reply:
Removed. It's unused for now. If it ever is needed, we can add it then.